packages/shortcode/src/index.js

```diff
@@ -143,9 +143,9 @@ export function string( options ) {
  *
  * @return {RegExp} Shortcode RegExp.
  */
-export const regexp = memize( ( tag ) => {
+export const regexp = ( tag ) => {
```

Comment:
Minor: Simply for consistency, we could format this as the other plain functions in this file now:

```js
export function regexp( tag ) {
	 // ...
}
```

Reply:
> Simply for consistency, we could format this as the other plain functions in this file now:

Good point, 1bf5ba0b7